src/test/java/org/candlepin/controller/OwnerManagerTest.java

```diff
@@ -34,70 +29,56 @@
 public class OwnerManagerTest {
-    @Mock private PoolManager mockPoolManager;
-    @Mock private ConsumerCurator mockConsumerCurator;
-    @Mock private ActivationKeyCurator mockActivationKeyCurator;
-    @Mock private EnvironmentCurator mockEnvironmentCurator;
-    @Mock private ExporterMetadataCurator mockExporterMetadataCurator;
-    @Mock private ImportRecordCurator mockImportRecordCurator;
-    @Mock private PermissionBlueprintCurator mockPermissionBlueprintCurator;
-    @Mock private OwnerProductCurator mockOwnerProductCurator;
-    @Mock private OwnerContentCurator mockOwnerContentCurator;
-    @Mock private OwnerCurator mockOwnerCurator;
-    @Mock private UeberCertificateCurator mockUeberCertificateCurator;
-    @Mock private OwnerServiceAdapter mockOwnerServiceAdapter;
-    @Mock private EventSink mockEventSink;
+    @Mock
+    private PoolService poolService;
+    @Mock
+    private ConsumerCurator mockConsumerCurator;
+    @Mock
+    private ActivationKeyCurator mockActivationKeyCurator;
+    @Mock
+    private EnvironmentCurator mockEnvironmentCurator;
+    @Mock
+    private ExporterMetadataCurator mockExporterMetadataCurator;
+    @Mock
+    private ImportRecordCurator mockImportRecordCurator;
+    @Mock
+    private PermissionBlueprintCurator mockPermissionBlueprintCurator;
+    @Mock
+    private OwnerProductCurator mockOwnerProductCurator;
+    @Mock
+    private OwnerContentCurator mockOwnerContentCurator;
+    @Mock
+    private OwnerCurator mockOwnerCurator;
+    @Mock
+    private UeberCertificateCurator mockUeberCertificateCurator;
 
     private OwnerManager createManager() {
         return new OwnerManager(
-            this.mockPoolManager, this.mockConsumerCurator, this.mockActivationKeyCurator,
+            this.poolService, this.mockConsumerCurator, this.mockActivationKeyCurator,
             this.mockEnvironmentCurator, this.mockExporterMetadataCurator, this.mockImportRecordCurator,
             this.mockPermissionBlueprintCurator, this.mockOwnerProductCurator, this.mockOwnerContentCurator,
-            this.mockOwnerCurator, this.mockUeberCertificateCurator, this.mockOwnerServiceAdapter,
-            this.mockEventSink);
+            this.mockOwnerCurator, this.mockUeberCertificateCurator);
     }
 
     @BeforeEach
     public void setup() {
         doAnswer(returnsFirstArg()).when(this.mockOwnerCurator).merge(any(Owner.class));
     }
 
-    @Test
-    public void testUpdateRefreshDate() {
-        Date initial = Util.yesterday();
-
-        Owner owner = new Owner();
-        owner.setLastRefreshed(initial);
-
-        OwnerManager manager = this.createManager();
-        Owner output = manager.updateRefreshDate(owner);
-
-        assertSame(output, owner);
-
-        assertNotNull(output.getLastRefreshed());
-        assertNotEquals(output.getLastRefreshed(), initial);
-        assertTrue(initial.before(output.getLastRefreshed()));
-    }
+    // TODO Test this class...
```

Comment:
Unfinished TODO?

Reply:
Kinda. There was a single test that I moved elsewhere. Instead of deleting the file I opted for a todo. I plan to add a few tests but probably as a separate PR (or card).